get-started/codelab/index.md

```diff
@@ -0,0 +1,1301 @@
+    The `divided` variable holds the final rows, converted to a list
+    by the convienice function, `toList()`.
+
+<!-- skip -->
+{% prettify dart %}
+  void _pushSaved() {
+    Navigator.of(context).push(
+      [[highlight]]new MaterialPageRoute([[/highlight]]
+        [[highlight]]builder: (context) {[[/highlight]]
+          [[highlight]]final tiles = _saved.map([[/highlight]]
+                [[highlight]](pair) {[[/highlight]]
+              [[highlight]]return new ListTile([[/highlight]]
+                [[highlight]]title: new Text([[/highlight]]
+                  [[highlight]]pair.asPascalCase,[[/highlight]]
+                  [[highlight]]style: _biggerFont,[[/highlight]]
+                [[highlight]]),[[/highlight]]
+              [[highlight]]);[[/highlight]]
+            [[highlight]]},[[/highlight]]
+          [[highlight]]);[[/highlight]]
+          [[highlight]]final divided = ListTile[[/highlight]]
+              [[highlight]].divideTiles([[/highlight]]
+            [[highlight]]context: context,[[/highlight]]
+            [[highlight]]tiles: tiles,[[/highlight]]
+          [[highlight]])[[/highlight]]
+              [[highlight]].toList();[[/highlight]]
+    );
+  }
+{% endprettify %}
+</li>
+
+<li markdown="1"> The builder property returns a Scaffold.
+    From the Material library, this widget provides an app bar for the
+    new route with the name, "Saved Suggestions". The body consists of a
+    ListView with the ListTiles.
+
+Add the highlisted code below:
+
+<!-- skip -->
+{% prettify dart %}
+  void _pushSaved() {
+    Navigator.of(context).push(
+      new MaterialPageRoute(
+        builder: (context) {
+          final tiles = _saved.map(
+                (pair) {
+              return new ListTile(
+                title: new Text(
+                  pair.asPascalCase,
+                  style: _biggerFont,
+                ),
+              );
+            },
+          );
+          final divided = ListTile
+              .divideTiles(
+            context: context,
+            tiles: tiles,
+          )
+              .toList();
+
+          [[highlight]]return new Scaffold([[/highlight]]
+            [[highlight]]appBar: new AppBar([[/highlight]]
+              [[highlight]]title: new Text('Saved suggestions'),[[/highlight]]
+            [[highlight]]),[[/highlight]]
+            [[highlight]]body: new ListView(children: divided),[[/highlight]]
+          [[highlight]]);[[/highlight]]
+        },
+      ),
+    );
+  }
+
+{% endprettify %}
+</li>
+
+<li markdown="1"> Hot reload the app. Favorite some of the selections and
+    tap the list icon in the app bar. The new route appears containing
+    the favorites. Note that the Navigator adds a "Back" button to the
+    app bar. You didn't have to explicitly implement Navigator.pop.
+    Tap the back button to return to the home route.
+</li>
+
+<li markdown="1"> You can easily change an app's theme by configuring
```

Comment:
Should changing the theme be a separate section, something that's on an equal footing with stuff like "Add Interactivity?"

Reply:
Yes, I can see done. Done.